src/main/webapp/table.js

```diff
@@ -139,10 +156,46 @@ class Table {
       }
       table.appendChild(rowElement);
     }
-    const pageInfoString = "Displaying rows " + rowToStart + " to " + rowToEnd + " of " + this.dataTable.length;
+    const pageInfoString = "Displaying rows " + rowToStart + " to " + rowToEnd + " of " + this.filteredRows.length;
     return pageInfoString;
   }
 
+  //Filters data by user input on the search bar
+  getFilteredRows(){
```

Comment:
I really like this method, the variable names and comments make it easy to follow!

Reply:
It's millennia's @hilakey !